### ISSUE TITLE
Multiple Packages Cannot Be Located

### ISSUE THREAD

**Issue by Milestacular** [LionSec/katoolin#202]
E: Unable to locate package vega
E: Package 'w3af' has no installation candidate
E: Unable to locate package http-tunnel
E: Unable to locate package magictree
E: Unable to locate package iphone-backup-analyzer
E: Package 'jd' has no installation candidate

I had to remove them from the apt install commands to allow the script to function correctly.

**Comment by gilgongo** [comment]
Weird - I was literally just going to post same issue as you just now. We must have been using katoolin at almost exactly the same time!

It seems a bit of a failing of the script that if it can't find packages it sends you back to the menu rather than installing what it can and notifying you to the missing ones.

**Comment by Milestacular** [comment]
> Weird - I was literally just going to post same issue as you just now. We must have been using katoolin at almost exactly the same time!
> 
> It seems a bit of a failing of the script that if it can't find packages it sends you back to the menu rather than installing what it can and notifying you to the missing ones.
> 
> EDIT: Seems a large number of packages are either missing or otherwise problematic with dependencies and things. I don't know if that means katoolin is unusable or not, but it seems like it :-(

I agree, tool isn't useable at the moment. I just ended up copying and pasting the commands from the script and removing the packages from the lines.

**Comment by gilgongo** [comment]
I see somebody' submitted a [PR that appears to be addressing this issue](https://github.com/LionSec/katoolin/pull/198), but they conclude the maintainer(s) have fled. Oh well.

**Comment by HashBandit** [comment]
> I see somebody' submitted a [PR that appears to be addressing this issue](https://github.com/LionSec/katoolin/pull/198), but they conclude the maintainer(s) have fled. Oh well.

So does that mean GG gameover for installing the kali tools in a quick and easy manner to ubuntu ?

**Comment by gilgongo** [comment]
Feels like it.

**Comment by HashBandit** [comment]
> Feels like it.

Ayy gg mon

**Comment by mkrupczak3** [comment]
Hey, I'll keep checking up on my fork if anyone wants to make PR's. I'm not much of a maintainer guy though so I would prefer to hand it off to someone else if possible.

Lionsec's website was dead and wasn't reminding to @ mentions, so I'm assuming this repo is dead.

**Comment by mkrupczak3** [comment]
Discussion on this repo being dead here:

https://github.com/LionSec/katoolin/issues/202#issuecomment-522265433

**Comment by mkrupczak3** [comment]
https://github.com/mkrupczak3/Katoolin-Robust

**Comment by tassiolucas** [comment]
You are a beautiful @mkrupczak3 !!! I do not believe that a soul of God solved this problem of a script with such a good idea but so poorly executed and left that way, with so many stars on GitHub and stopped like that !!! It's a shame what the administrators of this repository did in abandoning it like this! They could at least pass the administration to you! Since it was you who solved the problem! My most sincere congratulations! And thanks for the help !!!

**Comment by NibinNaug** [comment]
Thank you sooooo much, you're a genius ! ;)

**Comment by samueladams873** [comment]
Katolin is no longer available 
but there is a new Katoolin3 

https://ostechnix.com/install-kali-linux-tools-using-katoolin3-in-ubuntu-20-04-lts/[Katoolin3](https://ostechnix.com/install-kali-linux-tools-using-katoolin3-in-ubuntu-20-04-lts/)
